docs/08-cicd.md

```diff
@@ -1,67 +1,93 @@
 
-|Workflow|Type-checking|Jest tests|Build for E2E tests|E2E tests|Delivered to production|
-|---|---|---|---|---|---|
-|commit|:heavy_check_mark:|:heavy_check_mark:|:x:|:heavy_check_mark:|:x:|
-|weekly|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:x:|:x:|
-|bi-weekly|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:x:|:heavy_check_mark:|
-|api-triggered|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:x:|-|
+* Get a CircleCI [Personal API Token](https://circleci.com/docs/2.0/managing-pi-tokens/).
+* Trigger a build using the tool [trigger-pipeline.sh](../.circleci/trigger-pipeline.sh).
+* For more information on how to use it, execute it without parameters to see usage information.
 
-Whether `api-triggered` is delivered to production depends on the supplied parameters.
+## Workflows
 
-## About the "api-triggered" Workflow 
+Several workflows exist for different purposes:
+* **commit**: Run for all commit of a PR to ensure good code quality and working code.
+* **weekly_development_delivery**: Delivers 'integreat-test-cms' builds to development (Testflight and Beta channel) once a week.
+* **bi_weekly_production_delivery**: Delivers 'integreat' builds to production.
+* **api_triggered_delivery**: [Manually triggerable](#triggering-a-delivery-using-the-ci) workflow which delivers either 'integreat-test-cms' builds to development
+ or 'integreat' builds to production.
 
-This workflow can be triggered using the tool `yarn trigger-pipeline` as described [below](#triggering-a-build-in-ci).
-Depending on the supplied parameters the build is released to production.
+See the table below for a more detailed overview:
 
-### Triggering a Build in CI
+|Workflow|Checks|E2E tests|Delivery|integreat build|integreat-test-cms build|malte build|
+|---|---|---|---|---|---|---|
+|commit|:heavy_check_mark:|:heavy_check_mark:|:x:|:x:|:heavy_check_mark:|:heavy_check_mark:|
+|weekly_development_delivery|:heavy_check_mark:|:x:|Development|:x:|:heavy_check_mark:|:heavy_check_mark:|
+|bi_weekly_production_delivery|:heavy_check_mark:|:x:|Production|:heavy_check_mark:|:x:|:heavy_check_mark:|
+|api_triggered_delivery|:heavy_check_mark:|:x:|Depends|Depends|Depends|:heavy_check_mark:|
 
-If you decide to do a non-scheduled automatic release of the app the easiest way is to trigger a build in CI/CD.
-First you need to get a ["Personal API token"](https://circleci.com/docs/2.0/managing-api-tokens/).
-Then you can trigger a build using the tool `yarn trigger-pipeline`. If you execute it without parameters you will see the usage.
+Steps executed if *Checks* is checked :heavy_check_mark::
+* Linting
+* Flow type checking
+* Unit testing with jest
+* Building the app
 
-
-# Fastlane
+## Fastlane
 
 Fastlane is a task-runner for triggering build relevant tasks. It offers integration with XCode and the Android SDK for building and delivering the app.
 
-## Setup of Fastlane
+### Fastlane Setup
```

Comment:
Are you sure no links break with this change?

Reply:
I think I adjusted all links.